packages/govuk-frontend/src/govuk/components/summary-list/summary-list.yaml

```
@@ -8,63 +8,78 @@ params:
         type: string
         required: false
         description: Classes to add to the row `div`.
-      - name: key.text
-        type: string
-        required: true
-        description: If `html` is set, this is not required. Text to use within each key. If `html` is provided, the `text` option will be ignored.
-      - name: key.html
-        type: string
-        required: true
-        description: If `text` is set, this is not required. HTML to use within each key. If `html` is provided, the `text` option will be ignored.
-      - name: key.classes
-        type: string
-        required: false
-        description: Classes to add to the key wrapper.
-      - name: value.text
-        type: string
-        required: true
-        description: If `html` is set, this is not required. Text to use within each value. If `html` is provided, the `text` option will be ignored.
-      - name: value.html
-        type: string
+      - name: key
+        type: object
         required: true
-        description: If `text` is set, this is not required. HTML to use within each value. If `html` is provided, the `text` option will be ignored.
-      - name: value.classes
-        type: string
-        required: false
-        description: Classes to add to the value wrapper.
-      - name: actions.classes
-        type: string
-        required: false
-        description: Classes to add to the actions wrapper.
-      - name: actions.items
-        type: array
-        required: false
-        description: Array of action item objects.
+        description: The key of each row.
```

As this is part of an object which represents a single row in the summary list, suggest there shouldn't be an 'each' here:

```suggestion
        description: The key for this row.
```

---

Yeah, this is another one. We say things like:

* The title of each section
* The text content of each section
* The HTML content of each section
* The value of the skip link’s `href` attribute
* The value of the link's `href` attribute…

But it varies quite a lot so no golden rule set already

---

😭 Added another Trello card: https://trello.com/c/bYRTm149/21-for-options-in-arrays-of-objects-sometimes-we-say-eg-title-for-this-section-sometimes-title-for-each-section-sometimes-title-for